inst/qml/RobustBayesianMetaAnalysis.qml

```diff
@@ -218,6 +218,11 @@ Form
 		name:					"cohensd_testType"
 		visible:				measures_cohensd.checked
 		radioButtonsOnSameRow:	true
+		onValueChanged:
+		{
+			input_N1.itemDoubleClicked(0)
```

Comment:
Let's not snake_case

Reply:
well, that resulted in 900 lines of changed code, I guess that this will require a proper on runtime review
(I didn't include this commit to stable because I think that it is an unnecessary change potentially breaking stuff in a bug fixes release...)

Reply:
Makes sense. Let's only do what's necessary for the current release.